docs(TraceView): Add documentation for span filters as panel options (#98721)
grafana/grafana: docs/sources/panels-visualizations/visualizations/traces/index.md

## Span filters

The **Span filters** options control the initial state of the span filters when the visualization loads, allowing you to customize your trace analysis view.

* **Find in trace** - Set the initial value to focus on spans relevant to your query.
* **Show matches only** - Toggle the switch on to display only spans that match the defined filter criteria. This helps simplify trace interpretation.
* **Show critical path only** - Toggle the switch on to highlight only the spans in the critical path, which helps identify performance bottlenecks and their impact on overall latency.
* **Service name** and **Service name operator** - Define a specific service or pattern to narrow analysis to spans related to particular services.
* **Span name** and **Span name operator** - Filter spans by name or pattern to focus on specific span types or processes.
* **Min duration** and **Max duration** - Set duration thresholds to exclude spans outside the desired time range.
* **Tags** - Add one or more tags to further refine the filtering criteria so only relevant spans are displayed.

> The options support variable interpolation, where you can set the service name to a variable `$var` and the visualization will replace it with the value for the variable named `$var` in the span filters. This applies to the service name, span name, duration fields and tags.

Reviewer: ```suggestion
The options support variable interpolation, where you can set the service name to a variable `$var` and the visualization will replace it with the value for the variable named `$var` in the span filters. This applies to the service name, span name, duration fields, and tags.
```

## Panel options

{{< docs/shared lookup="visualizations/panel-options.md" source="grafana" version="<GRAFANA_VERSION>" >}}

## Span filters

> The **Span filters** options control the initial state of the span filters when the visualization loads, allowing you to customize and streamline your trace analysis view for better clarity and efficiency.

Reviewer: Just want to stick to the facts a bit more here and not try to market the feature in the technical docs.

```suggestion
The **Span filters** options control the initial state of the span filters when the visualization loads, allowing you to customize your trace analysis view.
```

> How to use **Span filters**:
> 

Reviewer: ```suggestion
```

> * **Find in trace**: Set the initial value for the span search to focus on spans relevant to your query.

Reviewer: ```suggestion
* **Find in trace** - Set the initial value to focus on spans relevant to your query.
```

> * **Show matches only** toggle: Display only spans that match the defined filter criteria, simplifying trace interpretation.

Reviewer: ```suggestion
* **Show matches only** - Toggle the switch on to display only spans that match the defined filter criteria. This helps simplify trace interpretation.
```

> * **Show critical path only**: Highlight only the spans in the critical path to identify performance bottlenecks and their impact on overall latency.

Reviewer: ```suggestion
* **Show critical path only** - Toggle the switch on to highlight only the spans in the critical path, which helps identify performance bottlenecks and their impact on overall latency.
```

> * **Service name** and **Service name operator**: Define a specific service or pattern to narrow the analysis to spans related to particular services.

Reviewer: ```suggestion
* **Service name** and **Service name operator** - Define a specific service or pattern to narrow analysis to spans related to particular services.
```

> * **Span name** and **Span name operator**: Filter spans by name or pattern to focus on specific span types or processes.

Reviewer: ```suggestion
* **Span name** and **Span name operator** - Filter spans by name or pattern to focus on specific span types or processes.
```

> * **Min duration** and **Max duration**: Set duration thresholds to exclude spans outside the desired time range, aiding in performance optimization.

Reviewer: ```suggestion
* **Min duration** and **Max duration** - Set duration thresholds to exclude spans outside the desired time range.
```

> * **Tags**: Add one or more tags to further refine the filtering criteria, ensuring only relevant spans are displayed.

Reviewer: ```suggestion
* **Tags** - Add one or more tags to further refine the filtering criteria so only relevant spans are displayed.
```

> Note that you must refresh the visualization for most span filter changes to take effect. This ensures the latest filters are applied to the displayed data.

Reviewer: ```suggestion
```

> Note that you must refresh the visualization for most span filter changes to take effect. This ensures the latest filters are applied to the displayed data.

Author: So I remove that entire last sentence then

* **Min duration** and **Max duration** - Set duration thresholds to exclude spans outside the desired time range.
* **Tags** - Add one or more tags to further refine the filtering criteria so only relevant spans are displayed.

> The options support variable interpolation, where you can set the service name to a variable `$var` and the visualization will replace it with the value for the variable named `$var` in the span filters. This applies to the service name, span name, duration fields and tags.

Author: @adrapereira @imatwawana thank you for the review - last change
